dev/com.ibm.ws.microprofile.opentracing.jaeger_fat/publish/servers/jaegerServer3/server.xml

```diff
@@ -0,0 +1,30 @@
+<server description="jaegerServer">
+
+    <featureManager>
+        <feature>mpOpenTracing-1.3</feature>
+    </featureManager>
+
+    <include location="../fatTestPorts.xml" />
+
+    <application location="mpOpenTracing.war">
+        <classloader apiTypeVisibility="+third-party"
+            commonLibraryRef="jaeger" />
+    </application>
+
```

Comment:
Do you need `<httpEndpoint>`?

Reply:
jaegerServer3 has the same server.xml as jaegerServer1. Only one server is run at a time for the test case it's being used in, so it will use the default http port without needing `<httpEndpoint>`.

Reply:
The default ports are 9080 and 9443 for http and https.  Do we want to use 
```
bvt.prop.HTTP_default=8010
bvt.prop.HTTP_default.secure=8020
```

Reply:
Discussed as not required due to importing `fatTestPorts.xml`.